src/pages/ibc/existing-protocols.mdx

```diff
@@ -74,15 +74,17 @@ transfer. It is often used for additional functionality like
 
 When you send an ICS20 transfer as described above, you do not get any feedback
 on whether the transfer was successful or not and the destination does not get
-informed of its newfound wealth. To solve this problem, the [ADR-8 specification][adr-8]
-was created. On the source chain, it provides callbacks when an IBC packet was
-acknowledged or timed out. On the destination chain, it triggers callbacks when
-a packet is received.
+informed of its newfound wealth. To solve this problem, the [ADR-8
+specification][adr-8] was created. On the source chain, it provides callbacks
+when an IBC packet was acknowledged or timed out. On the destination chain, it
+triggers callbacks when a packet is received.
```

Comment:
Is there a formatter script in this repo to get those changes myself next time?

Reply:
Not a formatter script, but I guess my vscode prettier extension kicked in, so `npx prettier`, I guess. But it looks like not everybody is using that anyways.